dodiscover/toporder/utils.py

```diff
@@ -0,0 +1,49 @@
+from typing import List
+
+import numpy as np
+from numpy.typing import NDArray
+
+
+def full_DAG(top_order: List[int]) -> NDArray:
+    """Find adjacency matrix of the fully connected DAG from the topological order.
+
+    Parameters
+    ----------
+    top_order : List[int]
+        Topological order of nodes in a causal graph.
+
+    Return
+    ------
+    A : np.ndarray
+        The fully connected adjacency matrix encoding the input topological order.
+    """
+    d = len(top_order)
+    A = np.zeros((d, d))
+    for i, var in enumerate(top_order):
+        A[var, top_order[i + 1 :]] = 1
+    return A
+
+
+def fullAdj2Order(A: NDArray):
```

Comment:
```suggestion
def full_adj_to_order(A: NDArray):
```

Follow snake-casing pattern in general